tests/php/test-class-amp-reader-themes-rest-controller.php

```diff
@@ -13,7 +13,7 @@
  *
  * @group reader-themes
  *
- * @covers AMP_Reader_Theme_REST_Controller
+ * @coversDefaultClass \AmpProject\AmpWP\Admin\ReaderThemes
```

Comment:
This seems to be wrong, the class that is tested is `AMP_Reader_Theme_REST_Controller` and the methods being tested are part of that, like `::get_items()`.

I suspect this was only meant to change the annotation type like so:
```suggestion
 * @coversDefaultClass AMP_Reader_Theme_REST_Controller
```